src/AccordionItem/accordion-item.spec.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import AccordionItemTitle from '../AccordionItemTitle/accordion-item-title';
+import AccordionItemBody from '../AccordionItemBody/accordion-item-body';
+import AccordionItem from './accordion-item';
```

Comment:
It seems like all of the component folders only contain the component, tests, and snapshots, so IMHO it's a bit overkill to have those folders. I also think those imports would be easier to parse if the file names were using the same case as the component names.